src/Surfnet/StepupSelfService/SelfServiceBundle/Tests/Service/GsspUserAttributeServiceTest.php

```diff
@@ -48,12 +48,6 @@ public function setUp(): void
         parent::setUp();
     }
 
-    public function tearDown(): void
```

Comment:
:+1: I removed the teardown also from RV but hadn't it backported yet. So thanks!